src/components/Layout/index.tsx

```diff
@@ -36,54 +37,56 @@ const Layout: FC<Props> = ({ children }) => {
+                    <div className={styles.header}>
+                        {navBarContent('Tasks', '/', router.pathname === '/')}|
+                        {navBarContent(
+                            'Issues',
+                            '/issues',
+                            router.pathname === '/issues'
+                        )}
+                        |
+                        {navBarContent(
+                            'Mine',
+                            '/mine',
+                            router.pathname === '/mine'
+                        )}
+                        {/* TODO: Uncomment when DS(Chanllenges) is ready */}
+                        {/* |
                     {navBarContent(
                         'DS',
                         '/challenges',
                         router.pathname === '/challenges'
                     )} */}
-                    |
-                    {navBarContent(
-                        'Open PRs',
-                        '/openPRs',
-                        router.pathname === '/openPRs'
-                    )}
-                    |
-                    {navBarContent(
-                        'Stale PRs',
-                        '/stale-pr',
-                        router.pathname === '/stale-pr'
-                    )}
-                    |
-                    {navBarContent(
-                        'Idle Users',
-                        '/idle-users',
-                        router.pathname === '/idle-users'
-                    )}
-                    {dev && (
-                        <>
-                            |
-                            {navBarContent(
-                                'Availability Panel',
-                                '/availability-panel'
-                            )}
-                        </>
-                    )}
-                </div>
+                        |
+                        {navBarContent(
+                            'Open PRs',
+                            '/openPRs',
+                            router.pathname === '/openPRs'
+                        )}
+                        |
+                        {navBarContent(
+                            'Stale PRs',
+                            '/stale-pr',
+                            router.pathname === '/stale-pr'
+                        )}
+                        |
+                        {navBarContent(
+                            'Idle Users',
+                            '/idle-users',
+                            router.pathname === '/idle-users'
+                        )}
+                        {dev && (
+                            <>
+                                |
+                                {navBarContent(
+                                    'Availability Panel',
+                                    '/availability-panel'
+                                )}
+                            </>
+                        )}
```

Comment:
- When using a component we don't call it as a function, it should rather be used as `<NavBarContent />` this helps react keep track of your component. 

- This flow is very hard to maintain and read what if we rather create an array containing all navigation links and the map over them to render the footer?
Along with this, we can have a function that tells us if we have to render this component or not given a particular set of conditions, this would help us conditionally render the link the same way you're doing. 
What do you think?

Reply:
That's correct. It can be written in a more readable way. 
But this was something already present and hence I did not fix it as part of this ticket. 
Do you want to fix it here? or create a new issue perhaps?

Reply:
- Even if we don't change the way the list is being rendered I would advise that we at least fix how the `navBarComponent` is being used here. And the rending of list can be taken up in a different PR.

Reply:
I have extracted navBarComponent as a Header component.